terra2/assetlist.json

```diff
@@ -1387,6 +1387,42 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
         }
       ]
+    },
+    {
+      "description": "Cosmo is the best currency in the universe.",
+      "denom_units": [
+        {
+          "denom": "ibc/7DD1E57984A26E7B6CBA0D93032DC24AD5274536184DFA999D166EBF6E4AA469",
+          "exponent": 0,
+          "aliases": [
+            "factory/inj1je6n5sr4qtx2lhpldfxndntmgls9hf38ncmcez/COSMO"
+          ]
+        },
+        {
+          "denom": "COSMO",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
```

Comment:
for ibc assets, please include the `traces` property, pointing back to injective

Reply:
added